### ISSUE TITLE
Manifest merging issue

### ISSUE THREAD

**Issue by jaredsburrows** [ncapdevi/FragNav#1]
```
:processDebugManifest
/AndroidManifest.xml:11:13-40 Error:
    Attribute application@supportsRtl value=(false) from AndroidManifest.xml:11:13-40
    is also present at [com.ncapdevi:frag-nav:1.0.2] AndroidManifest.xml:14:9-35 value=(true).
    Suggestion: add 'tools:replace="android:supportsRtl"' to <application> element at AndroidManifest.xml:8:5-25:19 to override.

See http://g.co/androidstudio/manifest-merger for more information about the manifest merger.

:processDebugManifest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':processDebugManifest'.
> Manifest merger failed : Attribute application@supportsRtl value=(false) from AndroidManifest.xml:11:13-40
    is also present at [com.ncapdevi:frag-nav:1.0.2] AndroidManifest.xml:14:9-35 value=(true).
    Suggestion: add 'tools:replace="android:supportsRtl"' to <application> element at AndroidManifest.xml:8:5-25:19 to override.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 1.963 secs
```

This forced me to add `android:supportsRtl="true"` to my app's manifest.

**Comment by ncapdevi** [comment]
Thanks for pointing this out. I'll push out a patch shortly.

**Comment by ncapdevi** [comment]
Fixed in 1.0.3
